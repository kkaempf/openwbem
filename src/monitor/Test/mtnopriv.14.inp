open in|out|ate 402
  /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/in-out-exists-ate
  xxx
open in|out|ate 402
  /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/in-out-exists-ate
  xxx
