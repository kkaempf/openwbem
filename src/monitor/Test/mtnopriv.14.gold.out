User: owprovdr

open
Exception:
  type: PrivilegeManagerException
  msg:  CheckException: open: insufficient privileges

open
Exception:
  type: IPCIOException
